sdk/core/core-client-paging-rest/src/paginate.ts

```diff
@@ -114,7 +88,10 @@ function checkPagingRequest(response: PathUncheckedResponse) {
 /**
  * Gets for the value of nextLink in the body. If a custom nextLinkName was provided, it will be used instead of default
  */
-function getNextLink(body: Record<string, unknown>, paginateOptions: PaginateOptions = {}) {
+function getNextLink(
+  body: Record<string, unknown>,
```

Comment:
I'd prefer we change this to `unknown` and avoid casting to record in the caller. We can re-cast when checking for the next link instead of relying on an index signature.

Reply:
Done.